fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/factory/FHIRTermGraphFactory.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (C) Copyright IBM Corp. 2020
```

Comment:
Year?

Reply:
This was created in 2020 and hasn't changed since.